evalai/utils/challenges.py

```diff
@@ -667,3 +669,72 @@ def display_leaderboard(challenge_id, phase_split_id):
         pretty_print_leaderboard_data(attributes, results)
     else:
         echo(style("Sorry, no Leaderboard results found.", bold=True, fg="red"))
+
+
+def upload_annotations_file_with_presigned_url(challenge_phase_pk, file):
+    url = "{}{}".format(get_host_url(), URLS.get_annotation_file_presigned_url.value)
+    url = url.format(challenge_phase_pk)
+    headers = get_request_header()
+
+    try:
+        # Fetching the presigned url.
+        data = {"file_name": file}
+        response = requests.get(url, headers=headers, data=data)
+        if response.status_code is not HTTPStatus.OK:
+            response.raise_for_status()
+
+        response = response.json()
+
+        # Uploading the annotation file for the current phase to S3.
+        echo(
+            style(
+                "Your file is being uploaded...",
```

Comment:
change this to `Uploading the test annotations.....`? or you can change it to `Uploading the {phase_name} phase annotations...`